app/org/maproulette/controllers/api/TaskController.scala

```diff
@@ -731,14 +725,6 @@ class TaskController @Inject() (
 
     val action =
       this.actionManager.setAction(Some(user), new TaskItem(task.id), actionType, task.name)
-    // add comment if any provided
```

Comment:
Doesn't this drop the comment if it was provided?

Reply:
After discussion, it appears that this would sync the OSM Comment with the MapRoulette tasks (users would get a MR notification). However this use case hasn't been used in at least 2 years. We're in agreement to remove this.